research/conditional/utils/model_utils.py

```diff
@@ -377,6 +383,30 @@ def get_ff_layer(args):
             seqlen=args.cutoff,
             mix_whole_batch=args.mix_whole_batch,
         )
+    elif args.ff_mode == "cont_moe_common_weighted_parameters":
+        return_fn = lambda: ContinuousMoECommonWeightedParameters(
+            dm=args.dmodel,
+            dff=args.dff,
+            n_experts=args.n_experts,
+            group_size=args.group_size,
+            sparsity_dim=args.sparsity_dim,
+            temperature=args.temperature,
+            expert_size=args.expert_size,
+            use_opt_einsum=args.use_opt_einsum,
```

Comment:
Tą flagę pewnie możnaby kiedyś usunąć, choćby dla czystości. Niewiele zmienia jeśli dobrze pamiętam, i w szczególności nie ma potrzeby żeby to było konfigurowalne. Tylko na stałe. No, ale to nie jest kwestia na tą PRkę.